src/beanmachine/paic2/README.md

```diff
@@ -0,0 +1,26 @@
+Paic2 Installation
+
+Paic2 installation has been tested on macos 12.3.1. 
+I have been using LLVM Clang version 14 to compile.
+
+1. Install Conda, Mamba, Ninja, cmake, and LLVM version 14. LLVM artifacts can be found here: https://github.com/llvm/llvm-project/releases
+2. Navigate to root of beanmachine
+3. `mamba env create -f src/beanmachine/paic2/environment.yml`
+4. `mamba activate bean-machine-39`
+5. `pip install -v -e .[dev]`
+6. `export C_COMPILER={path to llvm clang}`
+7. `export CXX_COMPILER={path to llvm clang++}`
+8. If using a Mac, set the environment variable CPLUS_INCLUDE_PATH. For example, if you installed LLVM to /usr/local/compiler, you may set the environment variable to something like this: `export CPLUS_INCLUDE_PATH=/usr/local/compiler/clang+llvm-14.0.0-x86_64-apple-darwin/include/c++/v1:/Library/Developer/CommandLineTools/SDKs/MacOSX12.3.sdk/usr/include`
```

Comment:
copy build script set. There should be two paths: (1) relative to installation of llvm (same llvm as CXX_COMPILER) and `$(xcrun --show-sdk-path)/usr/include`